App.js

```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
 
 export default class App extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      currentPage: 'Login'
+      currentPage: 'Home',
+      currentLatitude: null,
+      currentLongitude: null
     }
   }
 
+  componentDidMount() {
+    this.watchID = navigator.geolocation.watchPosition(
+      position => {
+        const { latitude: currentLatitude, longitude: currentLongitude } = position.coords;
+        this.setState({
+          currentLatitude,
+          currentLongitude
+        });
+      },
+      error => console.log(error),
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 0, distanceFilter: 1 }
```

Comment:
These arguments are for watchPosition,
enableHighAccuracy is for GPS tracking vs wifi
timeout after 20s
maximumAge:0 is so it updates whenever possible without using cached value
distanceFilter:1 is tracking any change greater than 1m.

Reply:
Ok thanks